### DISCUSSION TITLE
Version of the Web Interface  does not match the firmware

### DISCUSSION THREAD

**Discussion by MartinK79** [jomjol/AI-on-the-edge-device#2657] [Q&A]
Hello all, 

for some experiments (just for fun), I have 2 ESP32 cam devices and I tried to configure AI on the edge to read the water meter. 

I installed version 14.0 on the first device and version 15.3 on the second device. 

After logging in on the devices, whichever one, I get the message:

The version of the web interface (?) does not match the firmware version (04f2f23)! It is recommended to keep both on the same version!

Both the firmware version and the rest of the files I got from the respective packages, here from this page. 

In addition, in the latest version I can not create a reference image at all. The software simply does not respond.  I can adjust the parameters, such as brightness, etc, click on the first button, but it does nothing.

Does anyone know this problem? What do I have to do to make it work ?

Martin

**Comment by friedpa** [comment]
Just upload (again) the ZIP File. In the ZIP all Files needed are included. This message appears when you upload only the firmware without updating the HTML Files.

**Comment by MartinK79** [comment]
Hey friedpa, 

it sounds good. 

Especially since I actually only loaded bootloader.bin, firmware.bin and partiton.bin onto the device via ESP32 tool. After that, I copied the content of the sdcard folder from zip to the micro-sd-card. 

To avoid misunderstandings, can I just copy the zip file to the SD card?

Many thanks,
Martin

**Reply by MartinK79** [reply]
I checked the content of the folders in the sdcard folder. There already exist a html folder with html files.

**Reply by caco3** [reply]
please follow the instructions and upload the whole zip file. No extraction on your side needed. The HTML files must match the firmware version.

**Comment by friedpa** [comment]
Yes, then all the files are loaded at once. The html Files must fit to the firmware version.
